cluster/plugin/local/plugin.go

```diff
@@ -248,9 +251,43 @@ func InfoNode(ctx context.Context, c *client.Client) (swarm.Node, error) {
 	return node, err
 }
 
-func InfoToJSON(sw swarm.Swarm, node swarm.Node) (string, error) {
+// InfoAMPCore returns the number of AMP core services
+func InfoAMPCore(ctx context.Context, c *client.Client) (int, error) {
+	var count int
+	services, err := c.ServiceList(ctx, types.ServiceListOptions{})
+	if err != nil {
+		return 0, err
+	}
+	for _, service := range services {
+		if strings.HasPrefix(service.Spec.Name, "amp") {
```

Comment:
would be safer to filter on "amp_". You should add a constant (coreStackName) and fmt.Sprintf("%s_", coreStackName).